tools/ippfind.c

```diff
@@ -336,18 +336,14 @@ main(int  argc,				/* I - Number of command-line args */
                                argv + i)) == NULL)
             return (IPPFIND_EXIT_MEMORY);
 
-          while (i < argc)
+          do
             if (!strcmp(argv[i], ";"))
-              break;
-            else
-              i ++;
-
-          if (i >= argc)
-          {
-            _cupsLangPrintf(stderr, _("ippfind: Expected semi-colon after %s."),
-                            "--exec");
-            show_usage();
-          }
+            {
```

Comment:
This logic is different than the original code. We look for a semicolon argument, and show the error if we don't find one.